examples/list-binding-pattern-in-match-statement/list_binding_pattern_in_match_statement.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+
+const string show = "show";
+const string remove = "remove";
+const string all = "*";
+
+function matchCommand(string[] commands) {
+    match commands {
+        // The list binding pattern below matches lists that contain two list items
+        // in which the first list item matches with the `show` constant.
+        [show, _] => {
```

Comment:
This is not the list binding pattern this is the list match pattern. I think what we intend to show from this BBE is `var list-binding-pattern` right? @gimantha @lasinicl WDYT?

Reply:
Yes, this is a match-pattern. We can use `var [show, _]`